translations/en.json

```
@@ -1025,6 +1025,7 @@
     "privacy-photos": "Profile Photo Privacy",
     "privacy-and-security": "Privacy and security",
     "privacy-policy": "Privacy policy",
+    "privacy-show-to": "People who have already seen your profile picture will continue to do so",
```

I wonder if a name like below would be more informative in the future.

```suggestion
    "privacy-show-to-warning": "People who have already seen your profile picture will continue to do so",
```
or

```suggestion
    "privacy-show-to-title": "People who have already seen your profile picture will continue to do so",
```